skada/_self_labeling.py

```diff
@@ -273,7 +274,7 @@ def _check_proba(self):
             )
 
     @available_if(_check_proba)
-    def predict_proba(self, X):
+    def predict_proba(self, X, **kwargs):
```

Comment:
You add the `**kwargs` in the definition of the function but forgot to give them to `self.base_estimator.predict_proba(X, **kwargs)`